api/features/models.py

```diff
@@ -970,3 +979,52 @@ def get_update_log_message(self, history_instance) -> typing.Optional[str]:
 
     def _get_environment(self) -> typing.Optional["Environment"]:
         return self.feature_state.environment
+
+
+class FeatureExport(models.Model):
```

Comment:
2 things here: 

 - I'd perhaps argue this should be an EnvironmentExport? 
 - I'd rather these models live in either `environments/import_export` (if we decide this is an _environment_ export rather than features, `features/import_export` or the existing `import_export` app.